recipes/timemory/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "timemory" %}
+{% set version = "3.2.1rc10" %}
```

Comment:
We'll need a release at the end or add a label to the package as we don't wont to have pre-releases on the main channel.

Reply:
Yeah, I just currently have been using the pre-release while I modified some build stuff (i.e. the first commit was using v3.2.0)